### PR TITLE
[GamePhysics] Fixed Chasles' Theorem DD

### PR DESCRIPTION
**Pull request by samm82**
- Updated the Chasles' Theorem DD with the correct label/refname 
- Added Wikipedia source to Chasles' Theorem DD
- Removed Chasles' Theorem TM

_Closes #1815_

### CODE REVIEW

**Comment by smiths** [comment]
@samm82, the move of Chasles' theorem looks good, except for some reason two assumptions have been added that were not there before:

- A: objectDimension
- A: dampingInvolvement

Chasles' theorem does not depend on these assumptions.  They shouldn't be referenced.  Moreover, if we do use assumptions, we shouldn't just list them, there should be text to justify/explain the assumption.

**Comment by smiths** [comment]
The changes to stable look good to me.  Thanks for removing the unnecessary assumptions @samm82.

File: code/stable/gamephysics/Website/GamePhysics_SRS.html [hunk 1]
```diff
@@ -1525,7 +1525,7 @@ <h3>Data Definitions</h3>
                       <th>Notes</th>
                       <td>
                         <p class="paragraph">
-                          The linear velocity <em><b>v</b><sub>B</sub></em> (m/s) of any point B in a rigid body (from <a href=#assumpOT>A: objectTy</a>) is the sum of the linear velocity <em><b>v</b><sub>O</sub></em> (m/s) of the rigid body at the origin (axis of rotation) and the resultant vector from the cross product of the rigid body's angular velocity <em>ω</em> (rad/s) and the displacement vector between the origin and point B, <em><b>r</b><sub>OB</sub></em> (m).
+                          The linear velocity <em><b>v</b><sub>B</sub></em> of any point B in a rigid body (from <a href=#assumpOT>A: objectTy</a>) is the sum of the linear velocity <em><b>v</b><sub>O</sub></em> of the rigid body at the origin (axis of rotation) and the resultant vector from the cross product of the rigid body's angular velocity <em>ω</em> and the displacement vector between the origin and point B <em><b>r</b><sub>OB</sub></em>.
```
Comment (samm82) [inline code/stable/gamephysics/Website/GamePhysics_SRS.html]: The Notes section here essentially restates the equation. Is this desired? How much redundancy is bad? @smiths

Comment (smiths) [reply code/stable/gamephysics/Website/GamePhysics_SRS.html]: Restating the equation isn't really necessary, but it doesn't bother me.  The words are sometimes helpful to readers, especially those that aren't strong with mathematical notation.  I don't think we always want the word version, but I also don't think we should delete all instances.  We can leave the Notes field for this DD as it is.

**Comment by samm82** [comment]
Based on the conversation at #1815, I think reverting this change would be the easiest way to go about fixing the refinement issue. How should we go about doing that? @JacquesCarette

**Comment by JacquesCarette** [comment]
If the change was done in a single commit, `git revert` can do that.  Otherwise it's more complicated, and you'll have to google it.

**Pull request merged**